jerry-core/parser/js/js-lexer.c

```diff
@@ -974,6 +974,9 @@ lexer_parse_string (parser_context_t *context_p, /**< context */
       else if (*source_p == LEXER_NEWLINE_LS_PS_BYTE_1 && LEXER_NEWLINE_LS_PS_BYTE_23 (source_p))
       {
         source_p += 3;
+#if ENABLED (JERRY_ES2015)
+        length += 3 * raw_length_inc;
```

Comment:
Do we need this `raw_length_inc`, or can the raw length be computed as `source_p - string_start_p`. If yes, perhaps we could just remove the variable. This could be more future proof approach.

Reply:
IMO that way is not completely valid, since in raw strings there are character sequences like (\r\n) which going to be transformed (into \n) therefore a single subtraction is not enough.

Reply:
Yes, we could remove `raw_length_inc`, but we still would need another variable (which also has to be guarded with __JERRY_ES2015__) to check for __\r\n__ to __\n__ conversions, as __rerobika__ said. Also, `source_p` points to one character after literal ends when there is __$__ in string. So length (for raw string) would be `source_p - string_start_p - raw_length_dec`

@zherczeg I can add it to this commit, there is not so much to change, if you think this would improve performance or code readability :)